pokatlas.py

```diff
@@ -11,13 +13,16 @@ def __init__(self, atlas_path: pathlib.Path, img_name: str, img_size: str, img_f
         self.img_filter = img_filter
         self.repeat = repeat
         self.sprites = {}
+        self.sprite_hashes = {}
     
     def add_sprite(self, name, attributes):
         self.sprites[name] = attributes
 
     def get_sprites(self) -> dict:
         return self.sprites
     
+    def add_sprite_hash(self, name, hash):
+        self.sprite_hashes[name] = hash
```

Comment:
I would prefer moving this above `get_sprites()`.

Reply:
Done, I've moved it above `get_sprites()`. :)